test/integration_test_runner.rb

```diff
@@ -7,10 +7,10 @@ def setup
   end
 
   def test_local_mode
-    argv = '-r target/repositories/pmd -b master -bc config/design.xml ' \
-            '-p pmd_releases/6.1.0 -pc config/design.xml -l test/resources/project-test.xml'
+    argv = %w[-r target/repositories/pmd -b master -bc config/design.xml
+              -p pmd_releases/6.1.0 -pc config/design.xml -l test/resources/project-test.xml]
 
-    Cmd.execute("bundle exec bin/pmdtester #{argv}")
```

Comment:
Hm... The idea behind calling the shell script is, that we actually test the shell script, too. Otherwise, we don't have a integration test for the shell script, and we don't see, when we break the shell script...
We should have at least one test, that calls the PmdTester via the shell script. You could also create a separate integration test ruby script (like `integration_test_pmdtester.rb`).